libs/angular/src/directives/if-feature.directive.ts

```diff
@@ -23,7 +23,7 @@ export class IfFeatureDirective implements OnInit {
    * Optional value to compare against the value of the feature flag in the config service.
    * @default true
    */
-  @Input() appIfFeatureValue: FeatureFlagValue = true;
+  @Input() appIfFeatureValue: string | number | boolean = true;
```

Comment:
🎨 suggestion: this could import the available flag types instead of hardcoding them here, but maybe they don't change very often. The following is a bit hard to read, so if we do choose this route, maybe use an alias for the type instead

```suggestion
  @Input() appIfFeatureValue: FeatureFlagRuntimeTypes[FeatureFlag] = true;
```